packages/builder/src/components/common/DashCard.svelte

```diff
@@ -0,0 +1,63 @@
+<script>
+  import { Icon, Detail } from "@budibase/bbui"
+
+  export let title = ""
+  export let actionIcon
+  export let action
+  export let dataCy
+
+  $: actionDefined = typeof action === "function"
+</script>
+
+<div class="dash-card" data-cy={dataCy}>
+  <div
+    class={actionDefined ? "dash-card-header active" : "dash-card-header"}
+    on:click={() => {
+      if (actionDefined) {
+        action()
+      }
+    }}
```

Comment:
NAB: This can just be `on:click={action}`. If `action` is nullish, then nothing will happen. Just a bit terser!

Reply:
This has been updated